src/main/java/org/folio/sample/domain/mapper/BookMapper.java

```diff
@@ -15,4 +15,6 @@ public interface BookMapper {
 
   @Mapping(target = "id", ignore = true)
   Book fromDto(BookForCreationDTO source);
+
+BookDTO bookForCreationDTOToBookDTO(BookForCreationDTO bookForCreationDTO);
```

Comment:
Any reason this was added? I don't see that this is used

Reply:
Yea no reason, I added by mistake. I removed it now.